config.lua

```diff
@@ -70,6 +70,7 @@ kickIdlePlayerAfterMinutes = 15
 maxMessageBuffer = 4
 emoteSpells = false
 classicEquipmentSlots = false
+coinImagesURL = "http://some.url/images/store"
 
```

Comment:
Not sure what you mean.
But you are supposed to be able to change it, 25 is the amount you can send in one package (you can't transfer 24 or 26 to a player, you can only transfer / buy / sell in an increment of 25).

In OTs we might wanna change this, ex be able to transfer with an increment of 1 or if we use really high prices 1000 etc etc
And since you also added it to the source files you should add it to the config.lua file, even if it will default to 25 you should still make it easier to newbies to change it.

And id also add them to a new line like the house configs and maybe explain what they do, that what the URL should be etc etc